File: client-app/src/app/models/udhetari.ts
export interface Udhetari {
    id: string
    email: string
    password: string
    birthday: string
    emri: number
    mbiemri: string
  }

  export interface UdhetariuserFormValues{
    email: string;
    password: string;
    displayName?: string;
    userName?: string;
    emri?: string;
    mbiemri?: string;
    birthday?: string | Date;
  }
  

File: Domain/Udhetari.cs
using System;
using System.Collections.Generic;
using Microsoft.AspNetCore.Identity;

namespace Domain
{
    public class Udhetari : IdentityUser
    {
        public string DisplayName { get; set; }

        public DateTime Birthday { get; set; }
        public string Emri { get; set; }
        public string Mbiemri { get; set; }
        // public ICollection<Fluturimi> Fluturimet { get; set; }
    
    }
}